src/main/java/br/eti/kinoshita/testlinkjavaapi/TestCaseService.java

```diff
@@ -114,6 +116,18 @@ protected TestCase createTestCase(String testCaseName, Integer testSuiteId, Inte
         return testCase;
     }
 
+    public void updateTestCase(TestCase tc) throws TestLinkAPIException {

```

Comment:
The PHP method returns a mixed array.

On success:

    array('status_ok' => true, 'msg' => 'ok', 'operation' => __FUNCTION__);

And on error returns the default errors object.